### PR TITLE
Day 2

### PR DESCRIPTION
**Pull request by delbaoliveira**
Got my ☕. Let's do this 💪

### CODE REVIEW

**Comment by delbaoliveira** [comment]
Profile Page - User information 👇

**Comment by delbaoliveira** [comment]
Profile Page - Gallery component 👇

**Comment by delbaoliveira** [comment]
Login Modal 👇

**Comment by delbaoliveira** [comment]
JavaScript time 💃🏻 💃🏻

File: components/Header.less [hunk 1]
```diff
@@ -37,4 +37,10 @@
       .btn-style(48%, 5px, @purple, @white);
     }
   }
+
+  .login {
+    &:hover {
+      cursor: pointer;
```
Comment (Velsu) [inline components/Header.less]: It would be great if You added cursor pointer to other buttons as well 👍

**Comment by delbaoliveira** [comment]
Thanks for the the feedback @Velsu! 👍 Those buttons have been bugging me the whole day (haha, get it?), I'm planning on working on them tomorrow, and thanks for spotting that horizontal scroll bar. Didn't want that showing in the demo 😆

**Pull request merged**